src/bundles/pix_n_flix/functions.ts

```diff
@@ -610,8 +645,41 @@ export function set_fps(fps: number): void {
 }
 
 /**
- * Allows you to upload videos into Pix-n-Flix
+ * Sets the audio volume of the local video file played.
+ * Note: Only accepts volume video within the range of 0 to 100.
+ *
+ * @param volume Volume of video (Default value of 100)
+ */
+export function set_volume(volume: number): void {
+  enqueue(() => updateVolume(Math.max(0, Math.min(100, volume) / 100.0)));
+}
+
+/**
+ * Set pix-n-flix to use an uploaded local file
+ * as the video or image feed instead of the default camera feed.
+ */
+export function use_local_file(): void {
+  inputFeed = 'local';
+}
+
+/**
+ * Set pix-n-flix to use the image from given URL
+ * as the image feed instead of the default camera feed.
+ *
+ * @param URL URL of image to be loaded
+ */
+export function use_image_url(URL: string) {
+  inputFeed = 'imageURL';
+  url = URL;
+}
+
+/**
+ * Set pix-n-flix to use a video resource from given URL
+ * as the video feed instead of the default camera feed.
+ *
+ * @param URL URL of video to be loaded
  */
-export function use_local_video(): void {
```

Comment:
PR mentions implementing and testing `use_local_video`, though it seems to be about the local file / video URL feature